composer.json

```diff
@@ -240,7 +240,7 @@
         "e2e:web-update:prepare": " ",
         "e2e:web-update:start": " ",
         "ecs": "Checks all files regarding the PHP-CS-Fixer",
-        "ecs-fix": "Checks all files regarding the PHP-CS-Fixer and fixes them if possible",
```

Comment:
We should keep the old as alias because of shell history fast typing

Reply:
Ok @shyim I've reverted that changes. In fact is better ecs how you suggest. Thank you!